### PR TITLE
LPS-129260 Follow Up (2)

### PR DESCRIPTION
**Pull request by martamedio**
Previous PR: https://github.com/liferay-appsec/liferay-portal/pull/369

### CODE REVIEW

**Comment by martamedio** [comment]
ci:test:sf

**Comment by martamedio** [comment]
ci:test:relevant

**Comment by martamedio** [comment]
ci:test:relevant

**Comment by arthurchan35** [comment]
ci:test:relevant:nocompile

File: modules/apps/portal-crypto-hash/portal-crypto-hash-test/src/testIntegration/java/com/liferay/portal/crypto/hash/test/CryptoHashTest.java [hunk 1]
```diff
@@ -326,44 +324,134 @@ public void testCryptoHashVerifierWithStaticInput() throws Exception {
 						"JDJhJDEwJHVxZVh5YjF1dUdHZjZ2UWtvalljU08="))));
 	}
 
-	protected <S, R, E extends Throwable> R callService(
-			BundleContext bundleContext, Class<S> serviceClass,
-			String filterString, UnsafeFunction<S, R, E> unsafeFunction)
-		throws E {
+	private Configuration _addCryptoFactoryConfiguration(
```
Comment (arthurchan35) [inline modules/apps/portal-crypto-hash/portal-crypto-hash-test/src/testIntegration/java/com/liferay/portal/crypto/hash/test/CryptoHashTest.java:327]: Ahh I see. In the original test, Carlos had this method in base class, but named as createFactoryConfiguration:
https://github.com/liferay-appsec/liferay-portal/commit/6405aab7247c8ee90e9f8610c1f073510017725f#diff-6b57f70e9685d09699715780a7c1baaa34039e72a769d8a70e913f123dc7f775R203-R213
That's why we have two same name methods after merging base test to current test.

**Comment by arthurchan35** [comment]
ci:test:relevant:nocompile

**Comment by arthurchan35** [comment]
ci:test:sf

**Comment by arthurchan35** [comment]
Hey @martamedio @stian-sigvartsen 
I have made some coding standards improvements, please have a look.
If you recall anything else from Brian's comments, also please dont hesitate to add more improvements.

This is the last thing I do today.
I will on PTO tomorrow morning again, will be back in the afternoon!

Thanks

**Comment by martamedio** [comment]
@arthurchan35 I agree with your changes, but I think we need to improve the way of presenting the Base64 texts... I don't know if this is something that Carlos did by himself or if you have more information: May I ask you to search from which website we are getting the Base64 strings and the hash or encoding used in each of them?
I think knowing that I can work on present it in a different way, more readable.

Thanks!

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by martamedio** [comment]
ci:forward

**Pull request closed without merging**